src/components/summary-list/index.md.njk

```diff
@@ -2,65 +2,130 @@
+If you’re showing a mix of rows (where some rows include actions and some do not), add the `govuk-summary-list__row--no-actions` modifier class to the rows without actions. This is to ensure the bottom border is drawn correctly in some browsers.
 
 {{ example({group: "components", item: "summary-list", example: "mixed-actions", html: true, nunjucks: true, open: false}) }}
 
 ### Removing the borders
 
-The summary list includes some separating borders to help users read each row and its action.
+The summary list includes separating borders to help users by joining the information on each row and its action together.
+
+Think carefully before you remove row borders. Borders help many users find and read information that’s laid out in rows, especially users who zoom in on pages or use assistive technologies to magnify their screen.
 
 If your summary list does not have any actions, you can choose to remove the separating borders with the `govuk-summary-list--no-border` class.
 
 {{ example({group: "components", item: "summary-list", example: "without-borders", html: true, nunjucks: true, open: false}) }}
 
 To remove borders on a single row, use the `govuk-summary-list__row--no-border` class.
 
+## Summary cards
+
+
+If you’re showing multiple summary lists on a page, you can show each list within a summary card. This lets you visually separate each summary list and give each a title and some actions.
+
+Use summary cards when you need to show:
+ - multiple summary lists that all describe the same type of thing, such as people
+ - actions that will apply to all the items in a list
+
+Summary cards are often used in case working systems to help users quickly view a set of information and related actions.
+
+Do not use summary cards if you only need to show a small amount of related information. Use summary lists instead, and structure them with headings if needed.
+
+If you’re showing summary cards at the end of a longer journey, you might want to familiarise the user with them earlier on — such as when the user reviews individual sections.
+
+### Card titles
+
+Use the summary card’s header area to give each summary list a title.
+
+Each title must be unique and help identify what the summary list describes. For example, this could be the name of a specific person, organisation or professional qualification.
+
+Try to keep titles short and relevant. You can use one or two important values in the summary list — such as the first and last name of a person.
+
+{{ example({group: "components", item: "summary-list", example: "card-with-title", html: true, nunjucks: true, open: false}) }}
+
+### Adding card actions
+
+You can add card actions in the header, which will be shown after the summary card's title.
+
+For example, if every row action within the card to “change” something will take the user to the place, you can show a single “change” link as a card action. This helps avoid repeating the same row action on every row.
```

Comment:
This paragraph doesn’t quite read right. I think it’s missing ‘same’, but could also possibly be more concise. Something like:

```suggestion
For example, if every row action within the card will take the user to the same place, you can show a single “change” link as a card action instead. This helps avoid repeating the same action on every row.
```

Reply:
I agree with this but I think we still need to include "change" in the first sentence.

Reply:
Thanks both for spotting. How's this?

```suggestion
For example, if you have multiple rows with "change" actions that all take the user to the same place, you can show a single “change” card action instead. This helps avoid repeating the same row action on every row.
```

Reply:
Looks good to me!